### ISSUE TITLE
I did 'create contract' but...

### ISSUE THREAD

**Issue by RGMV** [ethereum/mist#2969]
- I did 'create contract' but it is still creating after 3 days...
That can't be good.

```
Version: `0.9.0`
OS & Version: Windows 7
Node version: `geth 1.6.6` 
Number of blocks synchronized: 4,164,378 (20-08-2017 / 19:41)
```
[http://www.verwimp.org/download/Test/Creating.jpg](url)
(Creating.jpg will be till september online)

**Comment by tomtom87** [comment]
@Rgmv I am having similar issues on ethereum network since last week, I cannot sync a wallet either. You are not alone. and yeh it's not good, this is not normal it was working before. Seems there is some network issues.

**Comment by RGMV** [comment]
Seems like somekind of loop.
He throws files downloaded away and download them again.
(at my computer at least.)

**Comment by tomtom87** [comment]
@RGMV yes just thrashes my hard disk, I left it on all night and my disk is in very poor condition now taking a long time to boot. I cannot use my machine while it is syncing either it maxes out IO

**Comment by RGMV** [comment]
Try defragmentation..
====

I open a cmd screen (i have windows 7) and use the parameters: 

cd\
cd C:\Users\User\AppData\Roaming\Ethereum Wallet\binaries\Geth\unpacked\

geth.exe --lightkdf --nousb --fast --cache 32
or
geth --lightkdf --nousb --fast --cache 32 console (and than start wallet)

**Comment by tomtom87** [comment]
@RGMV I tried this for 6 hours, same problems as before

> Maybe Ethereum has to create a database where we can download 'one on one' via fiber and then hook us to the sync net.

Yeh I been thinking this, like NEM has the db to download so you can sync in seconds.

**Comment by RGMV** [comment]
For newbees the database is just to big.. and it's getting bigger and bigger.
So the will to download (and wait and wait and wait) and keep sync is soon lost.
And the next thing what will happens is that you lose users. (customers)

And that my friend, is a bad thing..

**Comment by evertonfraga** [comment]
That behavior was fixed in 0.9.3. 

You can try the experimental `Develop > Sync with Light Client`.
